### ISSUE TITLE
Extend documentation 

### ISSUE THREAD

**Issue by drydenwilliams** [thegreenwebfoundation/co2.js#161]
**Is your feature request related to a problem? Please describe.**
There are lots of method names in the models that I find confusing. e.g `perByteTrace` and `perVisitTrace` within the [customise-website-carbon-calculations](https://developers.thegreenwebfoundation.org/co2js/tutorials/customise-website-carbon-calculations/) pages.

**Describe the solution you'd like**
With each method, it should have a description e.g. 

## `perByteTrace`
This means that it....

## `perVisitTrace`
This means that it....

**Comment by fershad** [comment]
Yep good point, it's worth having a small explainer (or link to an explainer) about each function.
